gerrydb/repos/column.py

```diff
@@ -163,16 +173,19 @@ def set_values(
     @online
     async def async_set_values(
         self,
-        path_or_col: Union[Column, str],
+        path: Optional[str],
         namespace: Optional[str] = None,
         *,
+        col: Optional[Column] = None,
         values: dict[Union[str, Geography], Any],
         client: Optional[httpx.AsyncClient] = None,
     ) -> None:
         """Asynchronously sets the values of a column on a collection of geographies.
 
         Args:
-            path_or_col: Short identifier for the column or a `Column` metadata object.
+            path: Short identifier for the column. Only this or `col` should be provided.
```

Comment:
same as above, pls clarify behavior for different combinations of path or col provided